src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -50,6 +50,13 @@
     public function getVersions();
 
     /**
+     * Get all version numbers where migration is in progress.
+     *
```

Comment:
`getVersions()` used to return all migrations, now it only returns completed migrations
`getPendingVersions()` will return migrations currently in progress.
`migrating()` will set the start time but not the end time (marking it as in progress).

For adapters other than mysql: `migrating()` does nothing and `pendingVersions()` will return an empty array, so that they keep their original behaviour